### PR TITLE
Updates t2t param

### PR DESCRIPTION
**Pull request by kwcantrell**
Fixes issue brought up in [here](https://urldefense.com/v3/__https://forum.qiime2.org/t/gemelli-phylo-rpca-with-taxonomy-overwriting-asv-taxonomy-in-an-unexpected-way/26059/7__;!!Mih3wA!AFuiYibvA7nIjV1zXp782Y0a6ruO6mZL5giAz8_mpOTvT5Ywo_mECLOyYNrJUKv-7NDwFB_4H32S82ZWKcKnJg$).

### CODE REVIEW

**Comment by cameronmartino** [comment]
Hi @kwcantrell,

Thanks for the PR. Could you move the version up [here](https://github.com/biocore/gemelli/blob/28d418438814fe2734e5b6c46b21080e1a81af4e/gemelli/__init__.py#L9)? Will want this patch to be the default install. 

Any comments @wasade?

File: gemelli/preprocessing.py [hunk 1]
```diff
@@ -217,16 +216,19 @@ def _get_taxonomy_io_stream(taxonomy):
     highest_rank_indx = taxonomy.dropna(axis=0, how='any').first_valid_index()
     rank_order = [rank[0]
                   for rank in taxonomy.loc[highest_rank_indx].values.tolist()]
-    nl.set_rank_order(rank_order)
 
-    # collapse taxonomy back into a single string
-    taxonomy.fillna('', inplace=True)
+    fill_missing_dict = {i: f'{r}__' for i, r in enumerate(rank_order)}
+    # collapse taxonomy_ back into a single string
+    taxonomy.fillna(value=fill_missing_dict, inplace=True)
```
Comment (wasade) [inline gemelli/preprocessing.py:222]: Does pandas index into `fill_missing_dict` by column name? I'm not familiar with this use

Comment (wasade) [reply gemelli/preprocessing.py:222]: Should a unit test assert what's going on with these changes?

File: gemelli/preprocessing.py [hunk 2]
```diff
@@ -93,7 +93,7 @@ def retrieve_t2t_taxonomy(phylogeny, taxonomy=None):
     counts = nl.collect_names_at_ranks_counts(tree_)
 
     nl.decorate_ntips(tree_)
-    nl.decorate_name_relative_freqs(tree_, counts, 2)
```
[the remark this answers is not in the document]
Comment (wasade) [reply gemelli/preprocessing.py:96]: I don't think so for the uses of gemelli

**Comment by wasade** [comment]
Sounds good re `fillna`!

File: gemelli/preprocessing.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (cameronmartino) [reply gemelli/preprocessing.py:96]: No need. We can keep it hard coded. Thanks!

**Comment by cameronmartino** [comment]
@wasade good to merge?

**Pull request merged**